guides/source/working_with_javascript_in_rails.md

```diff
@@ -63,7 +63,8 @@ import ReactDOM from "react-dom"
 ```
 
 Adding npm Packages with JavaScript Bundlers
---------
+--------------------------------------------
```

Comment:
This doesn't affect the output of the markdown, but it does look better when viewing the raw file, so this is fine. I wouldn't mention it as a fix though.